giskardpy/motion_statechart/tasks/pointing.py

```diff
@@ -80,73 +76,95 @@ def build(self, context: BuildContext) -> NodeArtifacts:
 
 @dataclass
```

Comment:
you need to use `@dataclass(eq=False, repr=False)`. this prevents dataclass from creating its own `__eq__` and `__repr__` implementation. the parent class has its own implementation for them